src/command_modules/azure-cli-billing/azure/cli/command_modules/billing/custom.py

```diff
@@ -20,3 +20,8 @@ def cli_billing_get_invoice(client, name=None):
 def cli_billing_list_periods(client):
     """List all available billing periods of the subscription"""
     return list(client.list())
+
+
+def cli_billing_list_enrollment_accounts(client):
+    """List all available enrollment accounts"""
+    return list(client.list())
```

Comment:
You do not need to wrap the SDK call with list.
The CLI should  handle this automatically.

Reply:
There's no custom code here either. Let me just replace this with a regular command that directly calls through to the underlying SDK.